src/chrome-storage-engine.spec.ts

```diff
@@ -90,7 +90,7 @@ describe('ChromeStorageStore', () => {
   it('should get entries', async () => {
     fakeStore[fakeStoreContentsKey] = JSON.stringify(mockEntries);
 
-    const entries = await chromeStore.getEntries();
+    const entries = await chromeStore.entries();
```

Comment:
Since we upgraded common SDK we also need to account for this `getEntries()` -> `entries()` rename for `IAsyncStore`